system/loggerd/loggerd.h

```diff
@@ -50,9 +33,11 @@ class EncoderInfo {
   int frame_height = 1208;
   int fps = MAIN_FPS;
   int bitrate = MAIN_BITRATE;
-  cereal::EncodeIndex::Type encode_type = cereal::EncodeIndex::Type::FULL_H_E_V_C;
+  cereal::EncodeIndex::Type encode_type = Hardware::PC() ? cereal::EncodeIndex::Type::BIG_BOX_LOSSLESS
+                                                         : cereal::EncodeIndex::Type::FULL_H_E_V_C;
```

Comment:
why was this changed?

Reply:
I see, ffmpeg encoder ignored the input, so it's not changed

Reply:
To remove the last arg codec from `constructor` . `FfmpegEncoder` ignore this arg, and hardcoded it to cereal::EncodeIndex::Type::BIG_BOX_LOSSLESS on pc:


>  FfmpegEncoder(const char* filename, CameraType type, int in_width, int in_height, int fps,
>                 int bitrate, cereal::EncodeIndex::Type **codec**, int out_width, int out_height,
>                 const char* publish_name) :
>                 VideoEncoder(filename, type, in_width, in_height, fps, bitrate, **cereal::EncodeIndex::Type::BIG_BOX_LOSSLESS**, out_width, out_height, publish_name) { encoder_init(); }
>